healthcare/api-client/v1/hl7v2/hl7v2_stores.py

```diff
@@ -0,0 +1,352 @@
+# Copyright 2018 Google LLC All Rights Reserved.
```

Comment:
```suggestion
# Copyright 2020 Google LLC All Rights Reserved.
```

Reply:
Not changing based on guidance from OSS.